mvrx/build.gradle

```diff
@@ -51,10 +60,20 @@ jacoco {
     toolVersion = "0.8.2"
 }
 
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion"
     implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:0.25.3'
```

Comment:
I believe you don't need to add `org.jetbrains.kotlinx:kotlinx-coroutines-core` as the dependency since it's added as the transitive dependency with `org.jetbrains.kotlinx:kotlinx-coroutines-android`